### PR TITLE
Added mocking support.

### PR DESCRIPTION
**Pull request by ElizabethOkerio**

### CODE REVIEW

File: src/Templates/ODataT4CodeGenerator.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿// ------------------------------------------------------------------------------
 // <auto-generated>
 //     This code was generated by a tool.
-//     Runtime Version: 16.0.0.0
+//     Runtime Version: 15.0.0.0
```
Comment (paulodero) [inline src/Templates/ODataT4CodeGenerator.cs]: user VS 2019

File: src/ViewModels/AdvancedSettingsViewModel.cs [hunk 2]
```diff
@@ -18,7 +18,7 @@ internal class AdvancedSettingsViewModel : ConnectedServiceWizardPage
         public string GeneratedFileName { get; set; }
         public bool IncludeT4File { get; set; }
         public bool MakeTypesInternal { get; set; }
-        public bool OpenGeneratedFilesInIDE { get; set; } = false;
+        public bool OpenGeneratedFilesInIDE { get; set; }
```
Comment (odero) [inline src/ViewModels/AdvancedSettingsViewModel.cs]: I take it this belongs to a separate PR

Comment (ElizabethOkerio) [reply src/ViewModels/AdvancedSettingsViewModel.cs]: yes. it is an update to the previous merged PR

Comment (KenitoInc) [reply src/ViewModels/AdvancedSettingsViewModel.cs]: Very weird. You should have done a simple fix on a separate PR.
